libpod/networking_linux.go

```diff
@@ -973,6 +973,13 @@ func (c *Container) getContainerNetworkInfo() (*define.InspectNetworkSettings, e
 		return nil, err
 	}
 
+	// If this a container with a netns and on the default network
+	if c.state.NetNS != nil && isDefault {
+		settings.Networks = make(map[string]*define.InspectAdditionalNetwork, len(networks))
+		cniNet := new(define.InspectAdditionalNetwork)
+		cniNet.NetworkID = c.runtime.netPlugin.GetDefaultNetworkName()
+		settings.Networks[c.runtime.netPlugin.GetDefaultNetworkName()] = cniNet
+	}
```

Comment:
This block is still unnecessary, though. All your work here is undone by the `if len(networks) > 0` block below.

Reply:
You also are going to need to modify the other use of `!isDefault` (L958) - the one that handles non-running containers.

Reply:
we should talk about that ... look at the block just beneath this. it looks there is step prior where the map is setup. lets discuss in watercooler.